src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDKSchemasV4.scala

```diff
@@ -7,6 +7,8 @@ object DetectLanguageResponseV4 extends SparkBindings[TAResponseV4[DetectedLangu
 
 object KeyPhraseResponseV4 extends SparkBindings[TAResponseV4[KeyphraseV4]]
 
+object SentimentResponseV4 extends SparkBindings[TAResponseV4[SentimentScoredDocumentV4]]
+
```

Comment:
There should be no changes for this file.

Reply:
run ```git checkout internship-ta -- src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDKSchemasV4.scala```

to remove these changes.

Reply:
Fixed, this object SentimentResponseV4 extends SparkBindings[TAResponseV4[SentimentScoredDocumentV4]]
was only moved to be under all of the previous objects. It was already there just moved up to the top of file.

Reply:
Ideally, this file should have been not checked in. Let's make sure we are only updating the files which are actually part of code change. Resolving this for now.